css-pseudo-4/Overview.bs

```diff
@@ -770,7 +812,8 @@ Backgrounds</h4>
   The ''::selection'' overlay is drawn
   over the ''::target-text'' overlay which is drawn
   over the ''::spelling-error'' overlay
-  which is drawn over the ''::grammar-error'' overlay.
+  which is drawn over the ''::grammar-error'' overlay which is drawn over the
+  ''::highlight'' overlays.
```

Comment:
The line breaks in this document aren't random, and aren't at 80ch, they're following https://rhodesmill.org/brandon/2012/one-sentence-per-line/

```suggestion
  which is drawn over the ''::grammar-error'' overlay
  which is drawn over the ''::highlight'' overlays.
```